controllers/landingPage.controller.js

```diff
@@ -0,0 +1,29 @@
+const mongoose = require('mongoose')
+const Categories = require("../models/categories.model");
+const Tutor = require("../models/tutor.model");
+
+
+const tutorsCardInfo = async (req, res) => {
+  try {
+    const getCategories = await Categories.find();
```

Comment:
can you explain me what are you gonna do with this for?

Reply:
I did refactor of landingPage rout in categories and tutors according with our conversation and it works much better. Thank you